apps/ui-library/components/side-navigation-item.tsx

```diff
@@ -108,11 +108,7 @@ const NavigationItem: React.FC<NavigationItemProps> = ({ item, onClick, ...props
         )}
       />
       {item.title}
-      {item.new && (
-        <Badge variant="success" className="capitalize">
```

Comment:
Sorry, just a minor, unrelated, bug I noticed and snuck in. The `capitalize` was doing the opposite, since we have that hardcoded on the component.